src/visible.js

```diff
@@ -102,10 +102,6 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				delay = -1;
 			}
 
-			if (typeof delay === "undefined") {
-				delay = -1;
-			}
-
```

Comment:
Hm, I previously thought using of typeof operator isn't clear.
But now I think it's not the problem.

Existing code tells a user can pass one argument, which is interpreted as 'delay' (in case of number type) or 'containment' (in case of other types).
e.g. this.inst.check(true); this.inst.check(); visible.check(200);

So I think existing code is okay.
But adding jsdoc for this situation, one argument is allowed, will be good.

Otherwise, below code can be clearer. :)
`
			if (typeof containment === "undefined") {
				containment = false;
				var type = typeof delay;
				if (type === "boolean") {
					containment = delay;
					delay = -1;
				} else if (type === "undefined") {
					delay = -1;
				} 
			}
`